features/step_definitions/common_steps.rb

```diff
@@ -20,6 +20,14 @@
   end
 end
 
+Given /^the response (?:(header ))?(.*) should be (.*)$/ do |header, attribute, value|
+  if header
+    expect(@response.headers[attribute.to_sym].to_s).to eq(value)
```

Comment:
This line seems odd to me. `.to_sym` converts a string to a symbol, which you then convert back to a string with `.to_s`. I think it should work as just `@response.headers[attribute].to_s`

Reply:
So the attribute arg _appears_ to need to be `to_sym`'d to be used as an accessor on the headers hash
Found it here:
https://stackoverflow.com/a/25538021/3967444

There may well be another way to do it 🤔

Reply:
Yup, you're right. I was reading the order of operations wrong. Head full of cotton wool today!